app/javascript/lib/shared/modules/events.js

```diff
@@ -0,0 +1 @@
+export const CREATE_DASHBOARD_EVENT = "create-dashboard-maker"
```

Comment:
Our fist event :crying_cat_face:

Reply:
would it make sense to think on namespacing it a bit? DASHBOARDS_CREATE_DASHBOARD_EVENT?

Reply:
Perhaps it's wordy right now (apart from this, there is only one else), but it's a good point. What about scope them?
```js
// events.js
export const GOBIERTO_DASHBOARDS = {
    CREATE: "create-dashboard-maker"
}
// file-using-events.js
import { GOBIERTO_DASHBOARDS } from "lib/shared/events"
document.addEventListener(GOBIERTO_DASHBOARDS.CREATE, () => {})
```

Reply:
lgtm

Reply:
Referenced in a new issue https://github.com/PopulateTools/gobierto/issues/3538. I'll face it after this PR